app/assets/stylesheets/common.scss

```diff
@@ -50,6 +50,10 @@ small, aside {
 time[title] {
   text-decoration: underline dotted;
 }
+time[title]:active::after,
+time[title]:focus::after {
+  content: " (" attr(title) ")";
```

Comment:
This isn't particularly i18n friendly, because it hard-codes the characters used for parenthesis and adds a space character, which might not be wanted in some languages e.g. japanese etc (I don't know for sure, but that's why we leave this to our translators, who are the experts!).

If you want to show some text that's not a perfect match for the existing title attribute, then it would be better to add the new text as a different attribute, and use the i18n system to construct that new display text.

I'm also concerned about whether this works properly for RTL text, but I haven't assessed that.

Overall, this feels like a "neat hack", but using CSS interactions to inject content into the text doesn't feel to me like a particularly robust method.

Reply:
The Japanese translated title looks a bit chatty. (I marked my parenthesis.)

![image](https://github.com/openstreetmap/openstreetmap-website/assets/2410353/9cb22f84-7420-4d6b-b2fe-fac8417ad059)
With datetime this would be:
![image](https://github.com/openstreetmap/openstreetmap-website/assets/2410353/c44ac7fc-7a2f-4595-a23f-90e6a88e9264)